### PR TITLE
[DS-3455] refactored out all mention of Elastic Search, code and configs,

### PR DESCRIPTION
**Pull request by hardyoyo**
removed ES dependency from the dspace-api pom.xml, added lucene-core 4.10.3 instead

### CODE REVIEW

**Comment by hardyoyo** [comment]
https://jira.duraspace.org/browse/DS-3455

**Comment by tdonohue** [comment]
At a glance, this LGTM 👍

**Review by tdonohue — APPROVED** [review]

File: dspace/config/modules/usage-statistics.cfg [hunk 1]
```diff
@@ -2,9 +2,7 @@
 #--------------USAGE STATISTICS CONFIGURATIONS------------------#
 #---------------------------------------------------------------#
 # These configs are only used by the DSpace interfaces which    #
-# track usage statistics (Solr or Elastic Search)               #
```
Comment (mwoodiupui) [inline dspace/config/modules/usage-statistics.cfg]: I think we need the first part of this line back.

**Review by mwoodiupui — APPROVED** [review]
+1 by inspection

**Review by terrywbrady — APPROVED** [review]

**Pull request merged**